sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -99,35 +99,44 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit c0c607c3a88933b4c5c20a0aca4539781c678411
-  # Merge: e18c691d 0f2d1f42
-  # Author: Omar Ahmed <omar.ahmed@codeplay.com>
-  # Date:   Wed May 29 15:02:39 2024 +0100
-  #     Merge pull request #1667 from nrspruit/fix_multi_device_event_cache
-  #     [UR] Fix Multi Device Event Cache for shared Root Device
-  set(UNIFIED_RUNTIME_TAG c0c607c3a88933b4c5c20a0aca4539781c678411)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/przemektmalon/unified-runtime.git")
```

Comment:
Assuming you update this appropriately before merging. This LGTM.

Reply:
Thanks for the help policing the UR repo/tag @JackAKirk. I've seen various reviewers do similar so I recently added the *Unified Runtime Updates* secetion to the [Contribute to DPC++](https://intel.github.io/llvm-docs/developer/ContributeToDPCPP.html#unified-runtime-updates) page. Note that the [intel/unified-runtime-reviewers](https://github.com/orgs/intel/teams/unified-runtime-reviewers) team is responsible for not approving PR's unless the repo/tag are correct.